Cargo.toml

```diff
@@ -33,6 +33,10 @@ tonic = { version = "0.8", default-features = false, features = [
 ] }
 tracing = "0.1"
 tracing-subscriber = "0.3"
+rustc-hex="^2.0.1"
```

Comment:
sort deps by alphabet

Reply:
https://crates.io/crates/cargo-sort introduce cargo sort in ci lint  check.

Reply:
Can you add this in this PR? It's not a big thing.

Reply:
Added
b94b2820e9552f00064f5b83dedaf0af7f633683